src/schema/objects/metadata.yaml

```
@@ -2825,6 +2825,57 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Distance between the participant's eye and the screen. If no screen was used, use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
+    - type: string
+      enum:
+        - n/a
+ScreenRefreshRate:
+  name: ScreenRefreshRate
+  display_name: Screen Refresh Rate
+  description: |
+    Refresh rate of the screen (when used), in Hertz.
+  type: number
+  unit: Hz
+ScreenResolution:
+  name: ScreenResolution
+  display_name: Screen Resolution
+  description: |
+    Screen resolution in pixel
+    (for example `[1920, 1200]` for a screen of 1920-length by 1080-height pixels),
+    if no screen use `n/a`.
+  anyOf:
+    - type: array
+      items:
+        type: integer
+        minItems: 2
+        maxItems: 2
+    - type: string
+      enum:
+        - n/a
+ScreenSize:
+  name: ScreenSize
+  display_name: Screen Size
+  description: |
+    Screen size in cm
```

as a clarification that only the "active screen" is meant?

```suggestion
    Screen size in cm, excluding potential screen borders
```